backend/app/instance_provider.go

```diff
@@ -59,11 +59,21 @@ func (ip *instanceProvider) GetKey(ctx context.Context, pluginContext backend.Pl
 }
 
 func (ip *instanceProvider) NeedsUpdate(_ context.Context, pluginContext backend.PluginContext, cachedInstance instancemgmt.CachedInstance) bool {
-	curSettings := pluginContext.AppInstanceSettings
-	cachedSettings := cachedInstance.PluginContext.AppInstanceSettings
-	return !curSettings.Updated.Equal(cachedSettings.Updated)
+	curConfig := pluginContext.Config
```

Comment:
I would call the new field `GrafanaConfig` to clarify that it's not the plugin config but the whole Grafana config